<!-- -*- mode: XML -*- -->
<launch>

    <node name="VisionFinder" pkg="finalProject2" type="car_vision.py" output="screen">
        <param name="sub_topic" type="string" value="/camera/color/image_raw" />
    </node>

</launch>

